GOVERNANCE.adoc

```diff
@@ -0,0 +1,71 @@
+independent link:https://github.com/apple/servicetalk/issues[github issues] should be opened prefixed with "[Vote]" and
+reference the proposal in question. The voting period should remain open for at least 2 weeks. Each Maintainer gets one
+vote and the majority will decide the direction. The final decision including the rational for the majority decision
+must be summarized by a maintainer.
+
+== Maintainers
+The ServiceTalk Maintainers are responsible for the long term maintenance of the ServiceTalk project/organization. Any
+additions to the repository are weighted on a spectrum of applicability and maintenance overhead. If a feature lies on
+the limited applicability and high maintenance end of the spectrum the maintainers tend to decide against expanding
+scope of the repository. In the future the maintainers may create a separate organization to facilitate discovery and
+collaboration for community projects. Until this time contributions can be maintained outside the ServiceTalk
+project, demonstrate maturity/adoption, and then be considered for contribution back to the ServiceTalk organization.
+
+=== Become a Maintainer
+Making contributions does not require becoming a maintainer, or obtaining commit access to the ServiceTalk
+project/organization. However if you are interested in taking a more formal role with decision making power you should
+consider becoming a Maintainer. Some characteristics of a Maintainer are as follows:
+
+* Be involved in contributing code, pull request review, triage of issues and addressing user questions in one or more
+forums such as Github, Stackoverflow, etc.
+* Maintain sustained contribution to the ServiceTalk project and spend a reasonable amount of time on it.
+* Show deep understanding of the areas contributed to, and good consideration of various reliability, usability,
+backward compatibility, and performance requirements.
+
+If you believe you satisfy these above criteria please open a
+link:https://github.com/apple/servicetalk/compare[github pull request] modifying the
+xref:MAINTAINERS.adoc[Maintainers doc] describing at least 5 non-trivial pull requests that have
+been accepted without major modifications. Please also make your case as to why you feel you will need Maintainer status
+moving forward and intention of continued involvement. Existing Maintainers must vote on the PR and a majority vote is
+required to merge and add the new Maintainer (see <<Decision Making and Voting>>).
+
+=== Expectations of a Maintainer
+After becoming a Maintainer you have the following expectations:
+
+* You are granted commit-after-approval to all parts of ServiceTalk.
+* You may commit an obvious change without first getting approval. The community expects you to use good judgment.
+Examples are reverting obviously broken patches, correcting code comments, and other minor changes.
+* You are allowed to commit changes without approval to the portions of ServiceTalk to which you have contributed or
```

Comment:
hmm... How much you need to have contributed before ? Or does this mean you can commit only changes to the portion of the code you contributed ?

Reply:
let me tighten up the "you can commit without approval" to require approval from 1 other maintainer.